sway-core/src/asm_lang/virtual_ops.rs

```diff
@@ -223,21 +223,575 @@ impl VirtualOp {
         .collect()
     }
 
```

Comment:
For all these giant functions I feel like there must be a more data driven compile time computational way to build these, with macros or something.  But I don't know specifically the best way.